### PR TITLE
TC-215, 222, 226, 227, 229-231

### PR DESCRIPTION
**Pull request by MariaMarkel**

### CODE REVIEW

File: debug.log [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[1113/212232.927:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```
Comment (vasylynash) [inline debug.log:1]: Remove this file

File: data/expected.json [hunk 2]
```diff
@@ -18,5 +18,10 @@
...
     "comedy": "Comedy"
   },
-  "labelCreate": "Create!"
+  "labelCreate": "Create!",
+  "storyPronoun": {
+    "his": "his",
+    "her": "her",
+    "its": "its"
+  }
 }
```
Comment (vasylynash) [inline data/expected.json:27]: Add empty line at the end of the file

File: data/selectors.json [hunk 3]
```diff
@@ -15,5 +15,6 @@
   "storyTitle": ".mb-4",
   "tryAgain": ".btn",
   "storyImage": ".error__pic",
-  "storyText": ".card-text"
-}
+  "storyText": ".card-text",
+  "moral": "//p[contains(text(), 'Moral')]"
+ }
```
Comment (vasylynash) [inline data/selectors.json:20]: Add an empty line at the end of the file

File: helpers/imageUpload.js [hunk 4]
```diff
@@ -1,13 +1,14 @@
+const sel = require("../data/selectors.json");
```
Comment (vasylynash) [inline helpers/imageUpload.js:1]: It looks like the same function already exists in the master

File: helpers/nameStartsWithCapital.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
+const sel = require('../data/selectors.json');
+const data = require('../data/testData.json');
+
+function isNameWithCapital () {
+    let storyText = $(sel.storyText).getText();
+    console.log(storyText);
```
Comment (vasylynash) [inline helpers/nameStartsWithCapital.js:6]: Remove console.log

File: helpers/nameStartsWithCapital.js [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+    let arr = storyText.split(".\n");
+    console.log(arr);
```
Comment (vasylynash) [inline helpers/nameStartsWithCapital.js:8]: Remove console.log

File: test/smoke/storyPage.js [hunk 7]
```diff
@@ -42,17 +42,18 @@ import { expect } from 'chai';
             const storyUrl = browser.getUrl();
             expect(appUrl).to.be.equal(storyUrl);
         });
+
```
Comment (vasylynash) [inline test/smoke/storyPage.js:45]: Remove empty line

File: test/smoke/storyPage.js [hunk 8]
```diff
@@ -42,17 +42,18 @@ import { expect } from 'chai';
...
             browser.url('');
             inputValues4(data.name, data.gender.she, data.age, data.storyType.Comedy)
-            imageUpload()
+            imageUpload(data.pathToFile)
             $(sel.create).click();
-            const tryAgain = $(sel.tryAgain).isDisplayed();
             browser.pause(3000);
+            const tryAgain = $(sel.tryAgain).isDisplayed();
             expect(tryAgain).to.be.equal(true);
         });
```
Comment (vasylynash) [inline test/smoke/storyPage.js:59]: Remove empty lines

File: test/smoke/storyToRead.js [hunk 9]
```diff
@@ -1,8 +1,11 @@
...
+const sel = require('../../data/selectors.json');
+const exp = require('../../data/expected.json');
+const data = require('../../data/testData.json');
+const inputValues4 = require('../../helpers/inputValues4');
+const inputValues4andClick = require('../../helpers/inputValues4andClick');
+const imageUpload = require('../../helpers/imageUpload');
+const isNameWithCapital = require('../../helpers/nameStartsWithCapital');
+import {expect} from 'chai';
 
 describe('Each type of the story uses the correct template', function () {
```
Comment (vasylynash) [inline test/smoke/storyToRead.js:11]: Remove empty lines

**Review by vasylynash — CHANGES_REQUESTED** [review]

File: debug.log (hunk 1 above)
Comment (MariaMarkel) [reply debug.log:1]: +

File: data/expected.json (hunk 2 above)
Comment (MariaMarkel) [reply data/expected.json:27]: done

File: data/selectors.json (hunk 3 above)
Comment (MariaMarkel) [reply data/selectors.json:20]: done

File: helpers/imageUpload.js (hunk 4 above)
Comment (MariaMarkel) [reply helpers/imageUpload.js:1]: it does!

File: helpers/nameStartsWithCapital.js (hunk 5 above)
Comment (MariaMarkel) [reply helpers/nameStartsWithCapital.js:6]: done

File: helpers/nameStartsWithCapital.js (hunk 6 above)
Comment (MariaMarkel) [reply helpers/nameStartsWithCapital.js:8]: done

File: test/smoke/storyPage.js (hunk 7 above)
Comment (MariaMarkel) [reply test/smoke/storyPage.js:45]: done

File: test/smoke/storyToRead.js (hunk 9 above)
Comment (MariaMarkel) [reply test/smoke/storyToRead.js:11]: removed

File: test/smoke/storyToRead.js [hunk 10]
```diff
@@ -24,7 +24,6 @@ describe('Each type of the story uses the correct template', function () {
             browser.url('');
             inputValues4(data.name, data.gender.she, data.age, data.storyType.Comedy);
             imageUpload(data.pathToFile);
-            //browser.pause(3000);
```
Comment (vasylynash) [inline test/smoke/storyToRead.js]: Remove commented line

**Review by vasylynash — CHANGES_REQUESTED** [review]

**Review by vasylynash — APPROVED** [review]

**Review by dannywhy — APPROVED** [review]

**Pull request merged**